### DISCUSSION TITLE
q-slider - use a font awesome icon as a label/slider

### DISCUSSION THREAD

**Discussion by eu16** [quasarframework/quasar#11467] [General - Components / Directives / etc] [answered]
Hi there

I am looking to replace the label of the q-slider using a font-awesome icon. Is there a way to do this? 

If so are you able to advise so?

I have tried to replace it using the label-value method but does not seem to work?

Thanks

**Comment by metalsadman** [comment, marked as answer]
You should file a feature request for such label slot, as for the slider there's a `thumb-path` prop but only accepts svg path string. do it with javascript for the mean time https://codepen.io/metalsadman/pen/jOLjXje

**Comment by alexandre-le-borgne** [comment]
Found a workaround while waiting for the slot

https://github.com/quasarframework/quasar/discussions/16777#discussioncomment-8142254
